File: Game/IntroSequence.cs
using System;

namespace Game
{
    public class IntroSequence
    {
        public IntroSequence() { }
        public void init()
        {
            Utility.ConsoleFunctions.cls();
            Console.WriteLine(@"
                                
                                
                                
                                
                                    Cardigan Games
                                
                                
                                
                ");
            Utility.ConsoleFunctions.wait(4000);
            Utility.ConsoleFunctions.cls();
            Console.WriteLine(@"
                                
                                
                                
                                
                                    The Card Game
                                
                                
                                
                ");
            Utility.ConsoleFunctions.wait(2000);
        }
    }
}

File: src/screens/IntroSequence.java
package screens;

import utility.Utility;

public class IntroSequence {
    public static void init( ) {
        Utility.ConsoleFunctions.cls( );
        System.out.println( """
                                
                                
                                
                                
                                    Cardigan Games
                                
                                
                                
                """ );
        Utility.ConsoleFunctions.wait( 4000 );
        Utility.ConsoleFunctions.cls( );
        System.out.println( """
                                
                                
                                
                                
                                    The Card Game
                                
                                
                                
                """ );
        Utility.ConsoleFunctions.wait( 2000 );
        MainMenu.init( );
    }
}
